include/sensors/core/tasks/pressure_driver.hpp

```diff
@@ -315,24 +315,32 @@ class MMR920 {
         can_client.send_can_message(
             can::ids::NodeId::host,
             can::messages::Acknowledgment{.message_index = count});
-        for (int i = 0; i < count; i++) {
+        std::array<uint32_t, 14> data{};
+        int i = 0;
+        while (i < count) {
```

Comment:
could make this `for(size_t i = 0; i < count; ) { }` if we wanted. in general, seems like we could use a size here instead of a signed value.